### PR TITLE
Fixing plugin routes

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Fixing plugin routes broken because the fake app removal

### CODE REVIEW

**Comment by jespino** [comment]
@lieut-data thanks you for the heads up :)

File: app/server_app_adapters.go [hunk 1]
```diff
@@ -139,6 +139,13 @@ func (s *Server) RunOldAppInitialization() error {
 	}
 	s.Router = s.RootRouter.PathPrefix(subpath).Subrouter()
 
+	// FakeApp: remove this when we have the ServePluginRequest and ServePluginPublicRequest migrated in the server
```
Comment (agnivade) [inline app/server_app_adapters.go:142]: Does this mean https://github.com/mattermost/mattermost-server/pull/14803 has to wait until this is fixed, or that can still go in ?

Comment (jespino) [reply app/server_app_adapters.go:142]: You can go in, this doesn't change anything, is just a bit ugly, and we have to take care of that. but nothing that is going to change anything fundamental.

**Comment by jespino** [comment]
@lieut-data can you QA the fix?

**Comment by lieut-data** [comment]
Looks good to me testing locally!

**Comment by lieut-data** [comment]
Skipping `3: QA Review` since tested by developers and this needs to be merged before tomorrow's `-daily` update.

**Comment by jespino** [comment]
Lets merge it to not break community-daily and make the QA after.

File: app/server_app_adapters.go (hunk 1 above)
Comment (agnivade) [reply app/server_app_adapters.go:142]: Filed https://mattermost.atlassian.net/browse/MM-26020.

**Pull request merged**